accord-core/src/test/java/accord/utils/README.md

```diff
@@ -0,0 +1,318 @@
+# Fuzz Testing
```

Comment:
in slack Jordan asked me to rename this `Property Testing` instead as `fuzz` is more common with security, and both interfaces documented are only "property" testing frameworks